### PR TITLE
Schema diff test

### PR DESCRIPTION
**Pull request by sagikazarmark**
⚠️ DO NOT SQUASH COMMITS ⚠️

### CODE REVIEW

File: internal/providers/google/gke_cluster_model.go [hunk 1]
```diff
@@ -85,7 +85,7 @@ type GKENodePoolModel struct {
 
 	Name             string `gorm:"unique_index:idx_cluster_id_name"`
 	Autoscaling      bool   `gorm:"default:false"`
-	Preemptible      bool
+	Preemptible      bool   `gorm:"default:0"`
```
Comment (orymate) [inline internal/providers/google/gke_cluster_model.go]:
```suggestion
	Preemptible      bool   `gorm:"default:false"`
```

File: internal/providers/oracle/objectstore_model.go [hunk 2]
```diff
@@ -36,7 +36,7 @@ type ObjectStoreBucketModel struct {
 
 	SecretRef string
 	Status    string
-	StatusMsg string
+	StatusMsg string `sql:"type:text;"`
```
Comment (orymate) [inline internal/providers/oracle/objectstore_model.go]:
```suggestion
	StatusMsg string `gorm:"type:text;"`
```

Comment (sagikazarmark) [reply internal/providers/oracle/objectstore_model.go]: Moved to #1600

File: internal/providers/google/gke_cluster_model.go (hunk 1 above)
Comment (sagikazarmark) [reply internal/providers/google/gke_cluster_model.go]: Thanks!

**Pull request merged**